ejb3/src/main/java/org/jboss/as/ejb3/timerservice/persistence/CalendarTimerEntity.java

```diff
@@ -23,11 +23,9 @@
 
 import java.lang.reflect.Method;
 import java.util.Date;
-
 import javax.ejb.ScheduleExpression;
 
 import org.jboss.as.ejb3.timerservice.CalendarTimer;
-import org.jboss.as.ejb3.timerservice.schedule.CalendarBasedTimeout;
```

Comment:
@chengfang Is this change related to primary key removal?

Reply:
this field and related method is not used anywhere, so it was removed as a cleanup, not related to the primary key removal.